stores/amazon_monitoring.py

```diff
@@ -133,62 +146,46 @@ def __init__(
         self.notification_handler = notification_handler
         self.check_shipping = checkshipping
         self.item_list: typing.List[FGItem] = item_list
-        self.stock_checks = 0
         self.start_time = int(time.time())
         self.amazon_config = amazon_config
         ua = UserAgent()
 
         self.proxies = get_proxies(path=PROXY_FILE_PATH)
+        ItemsHandler.create_items_pool(self.item_list)
 
         # Initialize the Session we'll use for stock checking
         log.debug("Initializing Monitoring Sessions")
         self.sessions_list: Optional[List[AmazonMonitor]] = []
-        for idx in range(len(item_list)):
-            connector = None
-            if self.proxies and idx < len(self.proxies):
-                connector = ProxyConnector.from_url(self.proxies[idx]["https"])
+        for idx in range(len(self.proxies)):
```

Comment:
This won't be suitable for merging unless it also supports the "default" configuration of no proxies.

Reply:
Also I don't really understand the structure of how this works. Instead of creating one AmazonMonitor for each item, you're now creating one for each proxy, and just cycling through the various items when assigning them to monitors?

It seems like this will only work if you have more proxies than items.

Reply:
But since you're cycling each `AmazonMonitor` through all the items, it seems like that shouldn't be hard to add, you'd just need to make sure that if there are no proxies, to instantiate a single `AmazonMonitor` that uses the default connector. 

This PR changes the meaning of the `delay` parameter from a per-item delay to a per-connection/proxy delay. That still seems totally workable, though, as long as people running the code are aware of it.

Reply:
Yeah I wrote it so that the delay parameter still works as intended - it's still the delay for check per session/proxy which should keep them getting dogg'd (503). I kinda figured if you're not using proxies alpha is kinda useless; might as well use master branch.

Reply:
I disagree pretty strongly that alpha is useless without proxies. I was getting softbanned using master because it was doing stock checks while logged in. Alpha uses anonymous sessions for stock checking, which is a huge improvement. I successfully bought what I wanted (granted, an AMD CPU, not a GPU) using alpha with no proxies.

Reply:
I've implemented a default mode with no proxies, though it looks ugly rn. Gonna try to clean it up once I figure out how.